miri-script/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+authors = ["Miri Team"]
+description = "Helpers for miri maintenance"
+license = "MIT OR Apache-2.0"
+name = "miri"
```

Comment:
If someone `cargo install`s this, unless they have a creative setup, this will stomp over the rustup proxy `miri` executable and break their install. The bin target here should be called something else, or otherwise prevent that situation.

Reply:
Great point, I'll rename binary to `miri-script` then.